Tests/SwiftFrameTests/ColorHexTests.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import XCTest
+@testable import SwiftFrameCore
+
+class ColorHexTests: XCTestCase {
+
+    func testCorrectColor() throws {
+        let color = try NSColor(hexString: "#ff0000")
+        XCTAssertEqual(color, NSColor.red)
```

Comment:
I think we should check that it's 255, 0, 0 because technically I think we have no guarantee that `NSColor.red` will be exactly that.